guides/_subsections/automated-analysis.md

```diff
@@ -65,8 +65,8 @@ Cryostat also provides an *Automated Analysis Card* that is able to display the
         </figcaption>
       </figure>
       <figure>
-        <a href="{{ site.url }}/images/2.4.0/automated-analysis-5.png" target="_blank">
-          <img src="{{ site.url }}/images/2.4.0/automated-analysis-5.png">
+        <a href="{{ site.url }}/images/2.3.0/automated-analysis-5.png" target="_blank">
+          <img src="{{ site.url }}/images/2.3.0/automated-analysis-5.png">
```

Comment:
Should these be referencing the new 2.4.0 images?

Reply:
No these are not needed in `2.4.0`  therefore should be removed( planned to do that on [#153](https://github.com/cryostatio/cryostatio.github.io/pull/153). I deleted the images in the `2.4.0` folder because we do not have the view report option just view in grafana.

Reply:
@andrewazores see the described changes above: [https://github.com/cryostatio/cryostatio.github.io/pull/153/commits/a8690f53e0c075b1219bf898187d624846121c79](https://github.com/cryostatio/cryostatio.github.io/pull/153/commits/a8690f53e0c075b1219bf898187d624846121c79)

Reply:
makes sense